packages/valory/agents/trader/aea-config.yaml

```diff
@@ -252,8 +252,6 @@ models:
       collateral_balance: ${int:10000000000000000000}
       mech_cost: ${int:10000000000000000}
       pool_fee: ${int:20000000000000000}
-      outcome_token_amounts: ${list:[11000000000000000000,9000000000000000000]}
-      outcome_token_marginal_prices: ${list:[0.4,0.6]}
```

Comment:
Did this leak to this PR somehow?

Reply:
No idea how this got removed, but its now back.